jcandane/bisum
# pytest framework and CI

- fix #2 and #8 
- Run tests with
  ```
   pytest
  ```
  or
  ```
   pytest -v
  ```
- run tests via GitHub actions on Linux, macOS, Windows for Python 3.8 - 3.11 (continuous integration)
- coverage is collected and sent to codecov.io (but still requires set up)
- check package build for every commit (continuous integration)
